src/main/java/frc/robot/drivetrains/simpletankdrivetrain/Robot.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.drivetrains.simpletankdrivetrain;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.spikes2212.command.drivetrains.TankDrivetrain;
+import com.spikes2212.command.drivetrains.commands.DriveArcade;
+import com.spikes2212.dashboard.RootNamespace;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.motorcontrol.MotorController;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
+public class Robot extends TimedRobot {
+
+    /**
+     * <p> A namespace is an object that holds values on a {@link NetworkTable}.</p>
+     * This is the main namespace which should host the main values and commands that don't belong to a single subsystem
+     * or command.
+     */
+    private final RootNamespace robotNamespace = new RootNamespace("robot");
+
+    /**
+     * This simple drivetrain has only 4 talons, 2 for each side. <br>
+     * The <b>ONLY</b> thing you need to change is the ports in the {@link RobotMap},
```

Comment:
change it to `the only things you may have to change are the ports... and the type...`